lib/CMakeLists.txt

```diff
@@ -112,8 +113,31 @@ else()
 	target_link_libraries(chiaki-lib OpenSSL::Crypto)
 endif()
 
-target_link_libraries(chiaki-lib Nanopb::nanopb)
-target_link_libraries(chiaki-lib Jerasure::Jerasure)
+if(CHIAKI_ENABLE_SWITCH AND NOT CHIAKI_ENABLE_SWITCH_LINUX)
+	# to provides csrngGetRandomBytes
+	target_link_libraries(chiaki-lib nx)
+endif()
+
+#if(CHIAKI_ENABLE_RPI)
+
+# Should set up some auto build of libilclient.
+
+	add_compile_definitions(OMX_SKIP64BIT)
+include_directories(/opt/vc/src/hello_pi/libs/ilclient
+	../pi)
+include_directories(include
+	/opt/vc/include/
+	/opt/vc/include/IL)
+target_link_libraries(chiaki-lib
+/opt/vc/src/hello_pi/libs/ilclient/libilclient.a
+	/opt/vc/lib/libbcm_host.so
+	/opt/vc/lib/libvcilcs.a
+	/opt/vc/lib/libvchiq_arm.so
+	/opt/vc/lib/libvcos.so)	
+#endif()
```

Comment:
Again, we need to have a define somewhere to enable these.
Where should that be set?